README.md

```diff
@@ -153,14 +153,15 @@ $ export no_proxy="localhost,127.0.0.1,localaddress,.localdomain.com,0.0.0.0,::1
 This is an open source project. We would love your help. We have prepared a [contributing](.github/CONTRIBUTING.md) guide to help you get started.
 
 If this is your first PR, or you're not sure where to get started,
-say hi in [Mycroft Chat](https://chat.mycroft.ai/) and a team member would be happy to mentor you.
-Join the [Mycroft Forum](https://community.mycroft.ai/) for questions and answers.
+say hi in [OpenVoiceOS Chat](https://matrix.to/#/!XFpdtmgyCoPDxOMPpH:matrix.org?via=matrix.org) and a team member would be happy to mentor you.
+Join the [Discussions](https://github.com/OpenVoiceOS/OpenVoiceOS/discussions) for questions and answers.
 
 ## Links
-* [Creating a Skill](https://mycroft-ai.gitbook.io/docs/skill-development/your-first-skill)
-* [Documentation](https://docs.mycroft.ai)
-* [Skill Writer API Docs](https://mycroft-core.readthedocs.io/en/master/)
-* [Release Notes](https://github.com/MycroftAI/mycroft-core/releases)
+* [Release Notes](https://github.com/OpenVoiceOS/ovos-core/releases)
+* [Mycroft Documentation](https://docs.mycroft.ai)
+* [Mycroft API Docs](https://mycroft-core.readthedocs.io/en/master/)
+* [OpenVoiceOS Chat](https://matrix.to/#/!XFpdtmgyCoPDxOMPpH:matrix.org?via=matrix.org)
```

Comment:
I'm already joined to all the OpenVoiceOS rooms, but does this link take someone to a room or list of rooms in the org? Would linking directly to `General` or `Development` be more helpful?

Reply:
i had the same question, this was taken from our website...

Reply:
I think if this repo is targeted at developers, I'd just link to the dev chat..